set.mm

```diff
@@ -33233,8 +33232,21 @@ Such interpretation is rarely needed (see also ~ df-ral ).  (Contributed
     vtoclg.1 $e |- ( x = A -> ( ph <-> ps ) ) $.
     vtoclg.2 $e |- ph $.
     $( Implicit substitution of a class expression for a setvar variable.
-       (Contributed by NM, 17-Apr-1995.) $)
+       (Contributed by NM, 17-Apr-1995.)  Avoid ~ ax-12 .  (Revised by SN,
+       20-Apr-2024.) $)
     vtoclg $p |- ( A e. V -> ps ) $=
+      ( wcel cv wceq wex elisset mpbii exlimiv syl ) DEHCIDJZCKBCDELPBCPABGFMNO
```

Comment:
Actually, you can remove even more axioms, see https://us.metamath.org/mpeuni/bj-vtoclg.html which removes ax-12, ax-ext and df-cleq (if you want to avoid the DV condition, use bj-elisset instead of bj-elissetv).